datagrid_gtk3/utils/transformations.py

```diff
@@ -105,6 +106,11 @@ def string_transform(value, max_length=None, oneline=True,
                 raise
             value = decode_fallback(value)
 
+    # Remove non-printable caracteres from the string. Only keep those
```

Comment:
haha np, better later than never :)

I use both en_US and pt_BR dictionaries on vim and when asking for it to correct the spelling, it gave the portuguese version a higher priority =P and I didn't notice because it made sense to me